console/console-init/ui/src/modules/iot-project-detail/Routes.tsx

```diff
@@ -6,7 +6,6 @@
 import React from "react";
 import { SwitchWith404, LazyRoute } from "use-patternfly";
 
-const getDevicesPage = () => import("modules/iot-device/DeviceListPage");
```

Comment:
how the page get rendered for device list?

Reply:
> how the page get rendered for device list?

@drcoolsanjeev We had duplicate routes for same. So i removed one of them.  For now we have this one.
const getDeviceListPage = () => import("modules/iot-device/DeviceListPage");